Add new SDK PHP version (#1175)
mercadopago/devsite-docs: guides/resources/changelog/current-year.en.md

## May 2021

### May 19th

> CHANGELOG
>
> New version of SDK PHP
>
> NEW_VERSION: NEW VERSION
>
> PRODUCT: SDK PHP

> We have released a new version of SDK PHP 1.11.0 and 2.3.0

Reviewer: ```suggestion
We have released a new version of SDK PHP 1.11.0 and 2.3.0.
```

> GIT
>
> GitHub
>

> > [Check GitHub](https://github.com/mercadopago/dx-php/releases/tag/1.11.0) for further details regarding this release.

Reviewer: ```suggestion
> [Check GitHub](https://github.com/mercadopago/sdk-php/releases) for further details regarding this release.
```

> > GIT

Reviewer: ```suggestion
```

> >

Reviewer: ```suggestion
```

> > GitHub

Reviewer: ```suggestion
```

> >

Reviewer: ```suggestion
```

> > [Check GitHub](https://github.com/mercadopago/dx-php/releases/tag/2.3.0) for further details regarding this release.

Reviewer: ```suggestion
```
